nextflow.config

```diff
@@ -222,15 +248,22 @@ manifest {
     name            = 'nf-core/proteinfamilies'
     author          = """Evangelos Karatzas""" // The author field is deprecated from Nextflow version 24.10.0, use contributors instead
     contributors    = [
-        // TODO nf-core: Update the field with the details of the contributors to your pipeline. New with Nextflow version 24.10.0
         [
             name: 'Evangelos Karatzas',
-            affiliation: '',
-            email: '',
-            github: '',
-            contribution: [], // List of contribution types ('author', 'maintainer' or 'contributor')
-            orcid: ''
+            affiliation: 'EMBL-EBI',
+            email: 'vangelis@ebi.ac.uk',
```

Comment:
Note: you don't need to include emails for everyone (to reduce spam), just the main contact is normally sufficient.